tests/bsim/bluetooth/host/adv/resume2/connectable/_build.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
Please place these _build and run.sh scripts inside a `test_scripts/` subfolder, that's the convention we have been following.